src/api/v1/motd.go

```diff
@@ -0,0 +1,16 @@
+package v1
+
+import (
+	"github.com/labstack/echo"
+	"net/http"
+)
+
+var motdText = "testing123"
+
+func init() {
+	// TODO load motd text from github
```

Comment:
doesn't this already exist in the current `/motd` code?

Reply:
ignore me i was thinking of the cloudflare worker xD

Reply:
Isn't that on cloudflare? I couldn't fine any motd stuff in `web`. This was only there to signal my intent to have motd in the api, it's implemented as a proxy in the latest push.

Reply:
yeah my bad, ignore me